dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -115,11 +115,22 @@ def main():
     collector_cls = config.Config.get_collector(collector['name'])
     collector_objects.extend(collector_cls.launch_collector(**new_args))
 
+  # Set global errors for final reporting
+  global_errors = []
+
   # Wait for collectors to finish and collect output
   collector_output = []
+  errors = []
   for collector_obj in collector_objects:
     collector_obj.join()
     collector_output.extend(collector_obj.results)
+    if collector_obj.errors:
+      err = (collector_obj.__class__.__name__, ", ".join(collector_obj.errors))
+      errors.append(err)
+      global_errors.append(err)
+      console_out.StdErr("ERROR:{0:s}:{1:s}\n".format(*err))
+  if errors:
+    pass  # pass or exit?
```

Comment:
It looks like you want this to be TODO here? Or remove this line all together, as it doesn't do anything.